controller/src/pid.cpp

```diff
@@ -116,16 +121,33 @@ void pid_execute(const VentParams &params, SensorReadings *readings) {
     break;
   }
 
+  //@TODO: The following are raw accesses that need to be updated to reference
+  // the new modules because currently they do not have the benefits of things
+  // like calibration, etc. that the module methods do. I didn't do this because
+  // I have no clue what the desired units are for the control system.
   // Update PID Loop
   int16_t sensorValue = Hal.analogRead(DPSENSOR_PIN); // read sensor
-  Input = map(sensorValue, 0, 1023, 0, 255);  // map to output scale
-  myPID.Compute();                            // computer PID command
+  // int16_t sensorValue = get_pressure_reading(PressureSensors::SomeDPPin);
+  Input = map(sensorValue, 0, 1023, 0, 255); // map to output scale
+  myPID.Compute();                           // computer PID command
   Hal.analogWrite(BLOWERSPD_PIN, static_cast<int>(Output)); // write output
 
   // Store sensor readings so they can eventually be sent to the GUI.
-  // TODO(jlebar): Is sensorValue actually pressure in cm of h2o?
   // TODO(jlebar): Get and store volume and flow data.
-  readings->pressure_cm_h2o = sensorValue;
+  // TODO(jlebar/verityRF): determine what volume_ml and flow_ml_per_min really
+  // means and how that is calculated from the two different venturi tubes in
+  // the air loop. Most likely need to have an integrator for flow to get the
+  // volume_ml.
+  // TODO(someone): is pressure_cm_h2o just the patient pressure sensor
+  // converted to the right unit? Convert [kPa] to [cm H2O] in place
+  readings->pressure_cm_h2o =
+      get_pressure_reading(PressureSensors::PATIENT_PIN) * 10.1974f;
   readings->volume_ml = 0;
-  readings->flow_ml_per_min = 0;
+  // example needs to be updated based on which of 2 DP sensors actually are
```

Comment:
Not sure what you mean; can you elaborate?

Reply:
https://docs.google.com/presentation/d/1Tin9kD9-UmLLGk1syUNVLj0kwz7nC2lE9lSnTD91C9c/edit#slide=id.g7330d84036_1_0 
Look at slide 9. What is flow (where) in regards to the air loop?

Reply:
Got it.  Elaborate in the comment?